xml_converter/src/string_helper.cpp

```diff
@@ -93,11 +118,6 @@ static const std::string base64_chars =
              "abcdefghijklmnopqrstuvwxyz"
```

Comment:
Since the functions here have been changed, should the attribution be removed?

Reply:
I dont believe I modified the `base64_encode` function so leaving the attribution there is fine it just only references the encode function now instead of both the encode and decode. Mildly confusing maybe but easily discoverable upon inquiry by clicking the link.